docs/components/dartpad/checkboxes/regular/main.dart

```diff
@@ -38,18 +38,21 @@ class _CheckboxesDemoState extends State<CheckboxesDemo> {
               Row(
                 children: [
                   Checkbox(
-                    onChanged: i == 4 ? null : (bool value) {
-                      setState(() {
-                        checked[i] = value;
-                      });
-                    },
+                    onChanged: i == 4
+                        ? null
+                        : (bool value) {
+                            setState(() {
+                              checked[i] = value;
+                            });
+                          },
                     tristate: i == 1,
                     value: checked[i],
                     activeColor: Color(0xFF6200EE),
                   ),
                   Text(
                     'Checkbox ${i + 1}',
-                    style: Theme.of(context).textTheme.subtitle1.copyWith(color: i == 4 ? Colors.black38 : Colors.black),
+                    style: Theme.of(context).textTheme.subtitle1.copyWith(
+                        color: i == 4 ? Colors.black38 : Colors.black),
```

Comment:
nit: would a trailing comma format this better?

Reply:
Done